### PR TITLE
[12.0] [MIG] project_task_send_by_mail

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Project Task Send By Mail
=========================
This module was written to extend the functionality of project tasks and allow you to send the task order by email.

### CODE REVIEW

File: project_task_send_by_mail/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "name": "Project Task Send By Mail",
+    "summary": "Send task report by email",
+    "version": "12.0.1.0.0",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/project",
+    "category": "Project Management",
+    "license": "AGPL-3",
+    "depends": [
+        "project_task_report",
```
Comment (HviorForgeFlow) [inline project_task_send_by_mail/__manifest__.py:14]: Should depends on project

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

File: project_task_send_by_mail/__manifest__.py (hunk 1 above)
Comment (Nikul-Chaudhary) [reply project_task_send_by_mail/__manifest__.py:14]: @hveficent No, we have used [report_template](https://github.com/OCA/project/pull/467/files#diff-5bc23089394fb44b1a07f1985af11e0dR11).

Here [project_task_report](https://github.com/OCA/project-reporting/tree/11.0/project_task_report)

Comment (HviorForgeFlow) [reply project_task_send_by_mail/__manifest__.py:14]: Then add oca dependencies

Comment (Nikul-Chaudhary) [reply project_task_send_by_mail/__manifest__.py:14]: @hveficent Yes, But it's (` project_task_report`) not Migrated in v12

File: project_task_send_by_mail/views/project_task_views.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<odoo>
+
+    <record id="view_task_form2" model="ir.ui.view">
+        <field name="name">project.task.form</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2"/>
+        <field name="arch" type="xml">
+            <field name="stage_id" position="before">
+                <button name="action_task_send" type="object" string="Send by Email" groups="base.group_user"/>
```
Comment (tarteo) [inline project_task_send_by_mail/views/project_task_views.xml:10]: Why the group here?

**Review by tarteo — CHANGES_REQUESTED** [review]

File: project_task_send_by_mail/views/project_task_views.xml (hunk 2 above)
Comment (Nikul-Chaudhary) [reply project_task_send_by_mail/views/project_task_views.xml:10]: @tarteo I did not add this group, It's already v11

**Comment by pedrobaeza** [comment]
@Nikul-Chaudhary can you review CIs status and comments?

**Comment by rafaelbn** [comment]
Hi , v12 has a new module https://github.com/odoo/odoo/tree/12.0/addons/digest maybe this could be a way to inform about task to the customers.

**Comment by Nikul-Chaudhary** [comment]
> @Nikul-Chaudhary can you review CIs status and comments?

@pedrobaeza waiting for `project_task_report`  [here](https://github.com/OCA/project-reporting/tree/11.0/project_task_report) 

If you Don't mind, Check Please: https://github.com/OCA/project/pull/467#discussion_r250299161

Cls: https://travis-ci.org/OCA/project/jobs/482989399#L1060

**Comment by pedrobaeza** [comment]
Then please migrate that module first.

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza Now It's Done
Can you please review and Move forward.

**Review by tarteo — APPROVED** [review]
LGTM

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza ready to merge

**Comment by pedrobaeza** [comment]
Please don't do merge operations. Rebase instead.

**Comment by Nikul-Chaudhary** [comment]
> Please don't do merge operations. Rebase instead.

@pedrobaeza Done

**Pull request merged**